src/main/java/core/basesyntax/service/impl/WriteServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriteService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class WriteServiceImpl implements WriteService {
+    private static final String REPORT_FILE_PATH =
+            "src/main/java/core/basesyntax/resources/report.scv";
+
+    @Override
+    public void writeReport(String report) {
+        try {
+            Files.writeString(Path.of(REPORT_FILE_PATH), report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t write to file: " + REPORT_FILE_PATH);
+        }
+    }
```

Comment:
It is better to get the path as parameter
```suggestion
    @Override
    public void writeReport(String filePath, String report) {
        try {
            Files.writeString(Path.of(filePath), report);
        } catch (IOException e) {
            throw new RuntimeException("Can`t write to file: " + filePath);
        }
    }
```